samples/psa/its/overlay-secure_storage.conf

```diff
@@ -7,4 +7,8 @@ CONFIG_MBEDTLS_PSA_CRYPTO_C=y
 # On top of that, the ITS implementation uses the stack for buffers.
 CONFIG_MAIN_STACK_SIZE=3072
 
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_NVS=y
```

Comment:
Could we use ZMS instead of NVS? Other places too.

Reply:
I guess it should be done over platform overlay where ZMS is suitable.

Reply:
@tomi-font, opinion?

Reply:
> Could we use ZMS instead of NVS? Other places too.

Soon: https://github.com/zephyrproject-rtos/zephyr/pull/81235